src/mococrw/asymmetric_crypto_ctx.h

```diff
@@ -408,6 +408,24 @@ class RSASignaturePublicKeyCtx : public DigestVerificationCtx, public MessageVer
     std::unique_ptr<Impl> _impl;
 };
 
+/**
+ * @brief Serialization formats for ECDSA signatures
+ */
+enum class ECSDASignatureFormat {
+    ASN1_SEQUENCE_OF_INTS, /**< Encoding of (r,s) as ASN.1 sequence of integers as specified in ANSI X9.62 */
+    IEEE1363, /**< Encoding of (r,s) as raw big endian unsigned integers zero-padded to the key length
+               *   as specified in IEEE 1363 */
+};
+
+/**
+ * @brief Serialization formats for ECDSA signatures (correctly spelled)
```

Comment:
remove the correctly spelled and rather put (with typo) to the other one

Reply:
done